src/util/qwlogging.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2024 WenHao Peng <near.kingzero@gmail.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#pragma once
+
+#include <qwglobal.h>
+extern "C" {
+#include <wlr/util/log.h>
+}
+
+QW_BEGIN_NAMESPACE
+
+void QW_EXPORT qw_log_callback(enum wlr_log_importance importance, const char *fmt, va_list args);
```

Comment:
How to use?

Reply:
`WServer::WServer(WServerPrivate &dd, QObject *parent)
    : QObject(parent)
    , WObject(dd)
{
#ifdef QT_DEBUG
    wlr_log_init(WLR_DEBUG, qw_log_callback);
#else
    wlr_log_init(WLR_INFO, qw_log_callback);
#endif
}
`   waylib will usage this

Reply:
No, you should export a `QWLog` class, and it's has a `init` static function.
We can use `qw_log_callback` function on default, but this function don't export to user.